holidays/groups/hindu.py

```
@@ -294,6 +360,19 @@ def _add_sharad_navratri(self, name) -> Optional[date]:
             name, self._hindu_calendar.sharad_navratri_date(self._year)
         )
 
+    def _add_tihar_holiday(self, name) -> Optional[date]:
+        """
+        Add Tihar Holiday (Nepal)
+
+        Tihar holiday is celebrated as the fifth day of Tihar festival, also known as Yamapanchak. 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)